### PR TITLE
feat(js): setup React Native

### PR DESCRIPTION
**Pull request by blu3beri**
Signed-off-by: blu3beri <blu3beri@proton.me>

### CODE REVIEW

File: build-xcframework.sh [hunk 1]
```diff
@@ -0,0 +1,189 @@
+#!/usr/bin/env sh
+
+NAME="aries_askar"
+VERSION="0.2.8-dev.1"
```
Comment (andrewwhitehead) [inline build-xcframework.sh:4]: Can we pull the version number from elsewhere to avoid having another place to update it?

**Comment by blu3beri** [comment]
It now uses jq to fetch the version from the `Cargo.toml`.

**Review by TimoGlastra — APPROVED** [review]
Is jq installed by default?

**Comment by blu3beri** [comment]
> Is jq installed by default?

Yeah that's the one thing I was not sure about. I will test it quickly on a random project.

**Comment by blu3beri** [comment]
@TimoGlastra yes jq is by default installed on GH runners :).

**Pull request merged**